jest.config.js

```diff
@@ -7,4 +7,5 @@ module.exports = {
   moduleNameMapper: {
     'react-query': '<rootDir>/src/index.ts',
   },
+  clearMocks: true,
```

Comment:
This option resets all calls for jest mock functions after every test, which is false by default.

It's important to set this to true, because otherwise `expect(x).toHaveBeenCalledTimes(1)` can return false positives because `x` was called in a previous test.